input/fsh/hiv-case-record.fsh

```diff
@@ -245,6 +260,7 @@ Description: ""
 * identifier.system = "http://openhie.org/fhir/hiv-casereporting/identifier/enrollment-unique-id"
 * managingOrganization 1..1 MS
 * diagnosis 1..1 MS
+* referralRequest MS  //Transfer-Out
```

Comment:
Shouldnt this field be mandatory? referralRequest 1..1 ?

Reply:
Why would every HIVEpisodeOfCare have a transfer out request? We did request the field cardinalities as well, but I'm not sure about making this mandatory.